docs/UserGuide.md

```diff
@@ -179,7 +183,7 @@ Format: `vclear`
 
 Volunteer Coordinators can create new events.
 
-Format: `ecreate n/EVENT_NAME r/ROLES_NEEDED… sd/START_DATETIME [ed/END_DATETIME] l/LOCATION dsc/DESCRIPTION [m/MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]`
+Format: `ecreate n/EVENT_NAME r/ROLES_NEEDED… sd/START_DATETIME [ed/END_DATETIME] l/LOCATION dsc/DESCRIPTION [m/QUANTITY MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]`
```

Comment:
perhaps can specify the quantity and what not under the restrictions instead of the format to reduce clutter since ppl have to see the restrictions anyway

Reply:
yeap okayy